asynq/futures.py

```diff
@@ -126,7 +126,13 @@ def _computed(self):
         # the notification even in case one of them fails.
         # Otherwise lots of negative effects are possible - e.g.
         # some of them might be left in blocked state.
-        self.on_computed.safe_trigger(self)
+        try:
+            self.on_computed.safe_trigger(self)
+
+        # pass on any exception in the callback, since we don't expect the caller to know how to
+        # deal with it
+        except Exception as e:
+            print('exception ignored in asynq on_computed callback: %s' % repr(e))
```

Comment:
Can you also print the stack trace with `traceback.print_exc()`?

Also, maybe document what happens when a BaseException is thrown. Does it mess up the scheduler?